libs/island-ui/core/src/lib/DatePicker/DatePicker.stories.tsx

```diff
@@ -9,17 +9,20 @@ import { DatePicker } from './DatePicker'
 export default {
   title: 'Form/DatePicker',
   component: DatePicker,
-  decorators: [withDesign],
-  parameters: withFigma({
-    desktop:
-      'https://www.figma.com/file/pDczqgdlWxgn3YugWZfe1v/UI-Library-%E2%80%93-%F0%9F%96%A5%EF%B8%8F-Desktop?node-id=50%3A155',
```

Comment:
Any reason the figma link has been removed?

Reply:
No real reason, just forgot to put it back in. There is something weird going on with the generated porps table, it won't list all the props from the interface only the currently used ones. I was thinking we need to do some debugging around this, at least it doesn't look like figma is causing the problem

Reply:
You need to use global import for React `import * as React` [here](https://github.com/island-is/island.is/pull/1731/files#diff-82d164b904bafdf368831dfb85e330a8cfae0df04ea86866271f1021d94081fbL1).

It's a very sad [issue](https://github.com/island-is/island.is/tree/master/libs/island-ui/storybook#props-issue) regarding the docgen plugin that pickup the props

Reply:
You're right! did not know that, thanks! 😄